mods/beds/functions.lua

```diff
@@ -60,8 +60,10 @@ local function lay_down(player, pos, bed_pos, state, skip)
 
 	-- stand up
 	if state ~= nil and not state then
+		if not beds.player[name] then
+			return
+		end
```

Comment:
Not sure if this is safe to do, if the overrides are missing it could just fall back to 1,1,1

Reply:
This is required, because when it's morning, players are kicked twice - the second kick being unneeded.